src/components/Snippet.jsx

```diff
@@ -46,11 +60,11 @@ class Snippet extends Component {
               In order to embed this content into your website or blog,
               simply copy and paste code provided below:
             </p>
-            <Input value="<script src='http://xsnippet.org/363686/embed/'></script>" />
+            <Input value={`<script src='http://xsnippet.org/${snippet.get('id')}/embed/'></script>`} />
           </div>
           <div className="snippet-code">
             <CodeMirror
```

Comment:
Shouldn't we somehow pass the snippet syntax to CodeMirror as well?

Reply:
Yes, we should. However, this is more challenging as we need to *import* proper syntaxes to enable highlighting. I'm working on this, so there will be a separate PR.